cordova-plugin-outline/apple/vpn/PacketTunnelProvider.m

```diff
@@ -283,8 +283,8 @@ - (NEPacketTunnelNetworkSettings *) getTunnelNetworkSettings {
                                                       networkPrefixLengths:@[@120]];
   ipv6Settings.includedRoutes = @[[NEIPv6Route defaultRoute]];
 
-  NEPacketTunnelNetworkSettings *settings = [[NEPacketTunnelNetworkSettings alloc]
-                                             initWithTunnelRemoteAddress:@"192.168.20.1"];
+  NEPacketTunnelNetworkSettings *settings =
```

Comment:
Seems worth mentioning that this is the address shown in the device's settings page?

Reply:
Good suggestion. Done in cc45b9b.